src/Compilers/CSharp/Portable/Binder/Binder_Crefs.cs

```diff
@@ -92,6 +92,12 @@ private NamespaceOrTypeSymbol BindNamespaceOrTypeSymbolInCref(TypeSyntax syntax)
             NamespaceOrTypeSymbol namespaceOrTypeSymbol = BindNamespaceOrTypeSymbol(syntax, unusedDiagnostics);
             unusedDiagnostics.Free();
 
+            // BindNamespaceOrTypeSymbol will wrap any tuple types in a TupleTypeSymbol. We unwrap it here, as doc comments don't consider the (T,T) form of tuples
+            if(namespaceOrTypeSymbol is TupleTypeSymbol t)
```

Comment:
I'm not sure were we stand on type-test vs. testing a property (`IsTupleType`). The type-test via pattern does seem convenient...
I don't feel strongly either way. I'll let others comment.

Reply:
We don't have access to IsTupleType, because it's a NamespaceOrTypeSymbol. Should I check if it's a TypeSymbol first, then check it's IsTupleType?

Reply:
Hum... I'm fine to leave as-is then.